packages/protocol/scripts/README.md

```
@@ -0,0 +1,44 @@
+arbitraries. Two demo artifacts ship alongside the test suite:
+
+## `demo.ts` — moat behavior walkthrough
+
+```bash
+bun run packages/protocol/scripts/demo.ts
+```
+
+Eleven scenarios, ~25 assertions. Each scenario prints what we tried, what the
+moat must do, and what it actually did. Sample output:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)